docs/man/man1/qtplasmac-materials.1

```diff
@@ -36,7 +36,7 @@ The file can be created by text entry or by importing text from a CAM tool file.
 .SH "SEE ALSO"
 See the QtPlasmaC section of the LinuxCNC Documentation for more information.
 
-http://linuxcnc.org/docs/2.9/html/plasma/qtplasmac.html#_material_converter
+http://linuxcnc.org/docs/stable/html/plasma/qtplasmac.html#_material_converter
```

Comment:
Including this change in the commit to merge forward to master fill cause a merge conflict, as the file has been removed in master.  You might want to consider handling it in a different commit.

Reply:
I will handle that as merge conflict.